### PR TITLE
(maint) Add Jekyll website

### PR DESCRIPTION
**Pull request by ekinanp**
Signed-off-by: Enis Inan <enis.inan@puppet.com>

### CODE REVIEW

**Comment by ekinanp** [comment]
You can view the website at `https://ekinanp.github.io/wash/`

File: docs/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.vagrant
+node_modules/*
+09-writing-advanced-plans/modules/stdlib
+_site
```
Comment (MikaelSmith) [inline docs/.gitignore]: Pretty sure this is the only line that's needed out of this file.

**Review by MikaelSmith — APPROVED** [review]
There are some lines we could remove from the gitignore. Also need to figure out the CI failure.

This and website can co-exist for a bit. Once we're ready to remove the old one, we'll delete `/website` (there may be some content there we still want to migrate elsewhere like the demo script) and remove the GitHub Actions that deploys it.

**Comment by MikaelSmith** [comment]
Rebase should fix the CI issue.

File: docs/.gitignore (hunk 1 above)
Comment (ekinanp) [reply docs/.gitignore]: Updated.

**Pull request merged**